packages/frontend/core/src/modules/workspace/entities/workspace.ts

```diff
@@ -48,6 +48,7 @@ export class Workspace extends Entity {
           },
           name: 'blob',
           readonly: false,
+          maxFileSize: this.engine.blob.maxBlobSize,
```

Comment:
i think this change is not good, because the maxBlobSize value will change

Reply:
use a `getter`